pyproject.toml

```diff
@@ -406,6 +406,7 @@ module = [
     "sentry.search.events.datasets.metrics",
     "sentry.search.events.datasets.metrics_layer",
     "sentry.search.events.datasets.profile_functions",
+    "sentry.search.events.datasets.profile_functions_metrics",
```

Comment:
new things should not be added to the typing blocklist -- please fix your annotations and get the type checker passing on this file